src/java/fr/paris/lutece/plugins/workflow/service/WorkflowProvider.java

```diff
@@ -361,6 +365,35 @@ public String getDisplayTasksForm( int nIdResource, String strResourceType, int
 
         return templateList.getHtml( );
     }
+    
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public String getDisplayProcessActionConfirmation( int nIdResource, String strResourceType, int nIdAction, Locale locale, String strTemplate )
+    {
+    	List<Integer> listIdResources = new ArrayList<>( );
+    	listIdResources.add( nIdResource );
+    	
+    	ResourceHistoryFilter filter = new ResourceHistoryFilter( );
+    	filter.setListIdResources( listIdResources );
+        filter.setResourceType( strResourceType );
+        filter.setIdAction( nIdAction );
+        
+        if ( CollectionUtils.isNotEmpty( _resourceHistoryService.getListHistoryIdByFilter( filter ) ) )
```

Comment:
Does this query on only get the last action result ?

Reply:
Not in all cases because we can assume the case where the action triggered by the user triggers another action in turn. This query therefore allows us to verify that the action triggered by the user is indeed traced in the history.